Wireframe/css/style.css

```diff
@@ -0,0 +1,80 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
+    background-color: #f4f4f4;
+    padding: 20px;
+}
+
+
+a{
+    padding: 14px;
+    background-color: #333;
+    font-weight: bold;
+    color: white;
+    margin-top: 20px;
+    text-decoration: none;
+}
+
+a:hover{
+    background-color: #635f5f;
+    color: hsl(0, 98%, 100%);
+    text-decoration: underline;
+}
+
+header {
+    text-align: center;
+    background-color: #333;
+    color: #fff;
+    padding: 20px 0;
+    margin-bottom: 20px;
+}
+
+.article2-article3{
+    display: flex;
+    justify-content: space-around;
+    gap: 100px;
+}
```

Comment:
rather than this approach, how would you use`:first-child pseudo-class` to apply unique styles to the first article?

Reply:
also, try some more unique styling for the first article! it should stand out from the others :)